packages/react-native/ReactCommon/react/renderer/components/switch/iosswitch/react/renderer/components/switch/IOSSwitchShadowNode.mm

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "IOSSwitchShadowNode.h"
+#import <UIKit/UIKit.h>
+
+namespace facebook::react {
+
+extern const char IOSSwitchComponentName[] = "Switch";
+
+#pragma mark - LayoutableShadowNode
+
+Size SwitchShadowNode::measureContent(
+    const LayoutContext& /*layoutContext*/,
+    const LayoutConstraints& layoutConstraints) const {
+      if (iosSwitchSize.width != 0){
```

Comment:
Why just checking the width? 🤔

Reply:
Just because it is to verify whether the struct has been initialized or not. Checking for both is redundant.

Width and height are 0 only if we never cached the values, so when the struct has just been created. Once the values are cached, it will never be 0 again.